File: pkg/houston/stage/stage.go
package stage

import (
	"os"
	"strings"
)

const (
	envStageKey = "STAGE"

	envProd = "prod"
	envDev  = "dev"
)

func IsProd() bool {
	return strings.ToLower(os.Getenv(envStageKey)) == envProd
}

func IsDev() bool {
	return strings.ToLower(os.Getenv(envStageKey)) == envDev
}

File: oracle/app/houston/stage.py
import os

ENV_STAGE_KEY = "STAGE"

ENV_PROD = "prod"
ENV_DEV = "dev"


def is_prod():
    return os.environ.get(ENV_STAGE_KEY).lower() == ENV_PROD


def is_dev():
    return os.environ.get(ENV_STAGE_KEY).lower() == ENV_DEV
